packaging/selinux/microshift.te

```diff
@@ -13,9 +13,6 @@ allow container_t container_runtime_tmpfs_t:file read_file_perms;
 allow container_t container_runtime_tmpfs_t:lnk_file read_file_perms;
```

Comment:
Removing the unwanted permissions is good, but have we confirmed that these other permissions (e.g. `allow container_t container_runtime_tmpfs_t:file read`), are actually being used? Given our investigations last week, I would have expected needing additional permissions (e.g. getattr) to be able actually read a file using standard libraries.

Reply:
It seems these aren't needed, in my local testing I can't seem to get audit logs to show denials when removing these, for now I'm removing them to keep us more inline with OCP